install_config/cluster_metrics.adoc

```diff
@@ -475,6 +475,78 @@ version corresponds to the very latest version available and can cause issues if
 newer version not meant to function on the version of {product-title} you are currently running.
 ====
 
+.Prometheus Variables
+[options="header"]
+|===
+
+|Variable |Description
+
+|`openshift_prometheus_state`
+|Set variable to install and update. Remove variable to uninstall.
+
+|`openshift_prometheus_namespace`
+|Project (namespace) where the components will be deployed.
+
+|`openshift_prometheus_replicas`
+|The number of replicas for prometheus deployment.
+
+|`openshift_prometheus_node_selector`
+|Selector for the nodes on which prometheus will be deployed.
+
+|`openshift_prometheus_image_<COMPONENT>`
+|Set to specify image for the component.
+
+|===
+
+*Prometheus Storage Related Variables*
+
+With each Prometheus component--including Prometheus,
+Alertmanager, alert-buffer, and oauth-proxy--you can set the PV claim by setting
+corresponding role variable, for example:
+
+----
+openshift_prometheus_storage_type: pvc
+openshift_prometheus_alertmanager_pvc_name: alertmanager
+openshift_prometheus_alertbuffer_pvc_size: 10G
+openshift_prometheus_pvc_access_modes: [ReadWriteOnce]
+----
+
+*Prometheus Alert Rules File Variable*
+
+You can add an external file with alert rules by setting the path to an additional
+rules variable:
+
+----
+openshift_prometheus_additional_rules_file: <PATH>
+----
+
+The file content should be in Prometheus alert rules format. The following
+example sets a rule to send an alert when one of the cluster nodes is down:
+
+----
+groups:
+- name: example-rules
+  interval: 30s # defaults to global interval
+  rules:
+  - alert: Node Down
+    expr: up{job="kubernetes-nodes"} == 0
+    annotations:
+      miqTarget: "ContainerNode"
+      severity: "HIGH"
+      message: "{{ '{{' }}{{ '$labels.instance' }}{{ '}}' }} is down"
+----
+
+*Prometheus Variables to Control Resource Limits*
+
+With each Prometheus component--including Prometheus, Alertmanager,
```

Comment:
Not certain if we have precedence in other docs but might be useful to reference the defaults/main.yaml in the repo so people can see the defaults.

Reply:
I linked to the associated README file.